documentation/source/conf.py

```diff
@@ -95,15 +94,15 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = dylan_themes.get_html_theme_default()
+html_theme = 'furo'
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
-html_theme_options = dylan_themes.get_html_theme_options_default()
+#html_theme_options = dylan_themes.get_html_theme_options_default()
 
 # Add any paths that contain custom themes here, relative to this directory.
-html_theme_path = [dylan_themes.get_html_theme_path()]
+#html_theme_path = [dylan_themes.get_html_theme_path()]
```

Comment:
It would be fine to just delete these lines, since we have the git history, but either way.

Reply:
Upps, sorry. Next time don't merge and I'll correct it.